MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,12 +20,18 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onErrorStateChanged: Observer<String?>?
```

Comment:
Suggestion:
`onErrorStateChanged` can be named `onErrorStateChange` (without `d` at the end) to be consistent with `onLoadingStateChange`
```suggestion
	var onErrorStateChange: Observer<String?>?
```